classes/workflow/EditorDecisionActionsManager.inc.php

```diff
@@ -130,24 +130,18 @@ static function _submissionStageDecisions() {
 				'operation' => 'externalReview',
 				'name' => 'externalReview',
 				'title' => 'editor.submission.decision.sendExternalReview',
-				'image' => 'advance',
-				'titleIcon' => 'modal_review',
+				'toStage' => 'editor.review',
```

Comment:
I believe that this is not needed here, as it is in OMP (when the submission is in the submission stage)? -- It seems to be used only if there are two steps, in promoteForm.tpl... Maybe later this decision action can be implemented in the same way, in two steps (first authors notification and then file selection)?

Reply:
I use the `toStage` parameter in the name of the second form part in `promoteForm.tpl`:

https://github.com/pkp/pkp-lib/pull/2733/files/0d17226923c0cf8ddc6954ac7deaa7418d53cf48#diff-033e78a65cd87eb957a71403cfa7a279R75